extras/check_version.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
 
+###
+# This script will check all source files containing the project version and exit with an error code -1 in case
+# they don't match.
+#
+# An optional argument is also accepted containing a version string to be checked agaisnt the others.
```

Comment:
Typo: ... against.

Reply:
Removed in https://github.com/HathorNetwork/hathor-core/commit/41d947719f42e32e6bb67b42ceb0b87e3608c732

Reply:
Thanks!